custom/readme_rx_printtofax.txt

```diff
@@ -1,25 +0,0 @@
-MI2 Print-to-Fax Option with signature and addendum/footer options
-
```

Comment:
If get no direction on this, would bump this to another PR request(then will spend some more time on analyzing the code where these globals are used to see if will be useful to have as global settings).